### PR TITLE
fix: list markdown line break

### PR DESCRIPTION
**Pull request by DragosIorgulescu**
## Preview

https://github.com/dailydotdev/apps/assets/1225100/902f214d-4cc4-44ec-bfbc-c0899934918e

## Events

_no events changed_

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [x] iOS (Chrome and Safari)
- [ ] Android

WT-1726 #done

### CODE REVIEW

File: packages/shared/src/components/RenderMarkdown.tsx [hunk 1]
```diff
@@ -128,6 +128,9 @@ const RenderMarkdown = ({
       linkTarget="_blank"
       remarkPlugins={plugins}
       components={{
+        ol({ children }) {
+          return <ol>{children}</ol>;
+        },
```
Comment (rebelchris) [inline packages/shared/src/components/RenderMarkdown.tsx:133]: Why is this needed, does it not already return it like this?

**Review by rebelchris — APPROVED** [review]
Looks ok, just not sure why it happens still.

File: packages/shared/src/components/RenderMarkdown.tsx (hunk 1 above)
Comment (DragosIorgulescu) [reply packages/shared/src/components/RenderMarkdown.tsx:133]: it does return the exact same thing without this, but for some reason explicitly setting this fixes the issue -- my hunch is that it has to do with re-renders of these components during streaming, but I could not tell you exactly why this solves it...

Comment (capJavert) [reply packages/shared/src/components/RenderMarkdown.tsx:133]: No idea why it would work but if it works let's role 🚀

**Review by capJavert — APPROVED** [review]
Let's role 🚀
